packages/build-essentials/src/webpack.config.js

```diff
@@ -33,8 +33,8 @@ const webpackConfig = {
                 loader: 'json-loader'
             },
             {
-                test: /\.(woff|woff2|eot|svg)$/,
-                loader: 'url-loader?limit=100000'
+                test: /\.(woff|woff2)$/,
```

Comment:
Current implementation of font-awesome.

https://github.com/neos/neos-ui/tree/57b0655ef67c52168c3298869b1a2906a711eefe/packages/react-ui-components/src/Icon/fontAwesome